packages/vx-demo/src/sandboxes/vx-area/Example.tsx

```diff
@@ -24,13 +33,13 @@ const getDate = (d: AppleStock) => new Date(d.date);
 const getStockValue = (d: AppleStock) => d.close;
 const bisectDate = bisector<AppleStock, Date>(d => new Date(d.date)).left;
 
-type Props = {
+export type AreaProps = {
```

Comment:
needed to export these for use with `GalleryTile`, now types will actually match 😄